app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -442,36 +442,42 @@ private boolean processIntentResponse(Intent response, boolean wasIntentCancelle
         return wasAnswerSet;
     }
 
-
     @Override
-    public boolean onPrepareOptionsMenu(Menu menu) {
+    public boolean onCreateOptionsMenu(Menu menu) {
         if (CommCareApplication.instance().isConsumerApp()) {
             // Do not show options menu at all if this is a consumer app
             return super.onPrepareOptionsMenu(menu);
```

Comment:
shouldn't this be onCreate now that it's moved?

Reply:
Thanks for catching.